microservices/playground_refresh/.env.example

```
@@ -0,0 +1,32 @@
+LOG_LEVEL=INFO
+
+# AWS
+AWS_REGION=us-east-1
+
+# Source (prod) RDS
+PROD_RDS_INSTANCE_ID=lokam-prod
+
+# Target (playground) RDS
+PLAYGROUND_RDS_INSTANCE_ID=lokam-playground
+PLAYGROUND_DB_INSTANCE_CLASS=db.t3.medium
+PLAYGROUND_DB_SUBNET_GROUP=default-vpc-0d0fcfe377a4c5d24
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PLAYGROUND_DB_SUBNET_GROUP key should go before the PLAYGROUND_RDS_INSTANCE_ID key

(UnorderedKey)